Armadillo.Agent/Program.cs

```diff
@@ -31,8 +32,13 @@ private static void ConfigureServices(IServiceCollection services)
                 builder.AddConsole(); 
             });
 
-            var credentials = new CredentialCache { { new Uri(ReportServerDataProvider.ReportServerUrl), "NTLM", CredentialCache.DefaultNetworkCredentials } };
-            var clientHandler = new HttpClientHandler { Credentials = credentials };
+            var builder = new ConfigurationBuilder()
+                    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                    .AddUserSecrets("28e6f711-a4c4-4cef-9e37-50ebfee35f91");
+            var configuration = builder.Build();
+
+            var clientHandler = new HttpClientHandler();
+            clientHandler.Credentials = new NetworkCredential(configuration["Username"], configuration["Password"]);
```

Comment:
Is it OK that now we don't mention NTLM anywhere? Will negotiate the authentication type automatically?

Also would be good to use default credentials as before if no Username/Password set in the app secrets.